src/test/java/org/broadinstitute/hellbender/utils/hmm/HMMUnitTest.java

```diff
@@ -807,6 +807,53 @@ public void testViterbiWithNullDataPoints() {
                 new ArrayList<>(Arrays.asList(1, 2, 3)), model);
     }
 
+    @Test
+    public void testGenerateHiddenStateChainOnUninformativeHMM() {
+        final int numStates = 3;
+        final UninformativeTestHMModel model = new UninformativeTestHMModel(numStates);
+
+        final int chainLength = 100000;
+        final List<Integer> positions = IntStream.range(0, chainLength).boxed().collect(Collectors.toList());
+
+        final List<Integer> chain = model.generateHiddenStateChain(positions);
+
```

Comment:
this is super nitpicky, but the white space here is not quite the same as in the second test...